app/tests.py

```diff
@@ -1,3 +1,123 @@
+        nature = Nature.objects.create(name='nt', code_leb='l5', code_dub='d5'),
+        trading_center = TradingCenter.objects.create(name='trct', code_leb='l6', code_dub='d6'),
+        nationality = Nationality.objects.create(name='nlty', code_leb='l7', code_dub='d7'),
+        deposit_place = DepositPlace.objects.create(name='Dpp', code_leb='l8', code_dub='d8'),
+        quotation_place = QuotationPlace.objects.create(name='QP', code_leb='l9', code_dub='d9'),
+        asset_allocation = AssetAllocation.objects.create(name='AsA', code_leb='l10', code_dub='d10'),
+        group_for_ledgers = '4211',
+        general_ledger = '4649',
+        provider_code = 'pro',
+        provider_ratelist = RateListProvider.objects.create(name='rlp', code_leb=11, code_dub=11),
+        multiplier_for_online_prices = 1,
+        isin = 'isi',
+        fixing = True,
+        fix1 = '10:00',
+        fix2 = '11:00',
+        mar_short_position = '11',
+        mar_lng_position = '12',
+        main_short_position ='13',
+        main_lng_position = '14',
+        moody = 'aaa',
+        fitch = 'bbb',
+        s_and_p = 'ccc'
+        )
+
+       
+        actual = sec.checkAllDropdownsOnBothLebDub()
+        expected = sec.save()
+
+
+
+        self.assertEquals(actual, expected)
+
+
+
+    def test_save_bond_2(self):
+
+       sec= SecurityBond.objects.create(
+       code= 'Test17',
+       circular='NULL',
+       designation= 'test17',
+       symbol = 'Test17',
+       currency = Currency.objects.create(name='cur', code_leb='l1', code_dub='d1'),
+       subtype= Subtype.objects.create(name='sb', code_leb='l2', code_dub='d2'),
+       category = Category.objects.create(name='ct', code_leb='l2', code_dub='d3'),
+       trading_category = TradingCategory.objects.create(name='tcat', code_leb='l4', code_dub='d4'),
+       nature = Nature.objects.create(name='nt', code_leb='l5', code_dub='d5'),
+       trading_center = TradingCenter.objects.create(name='trct', code_leb='l6', code_dub='d6'),
+       nationality = Nationality.objects.create(name='nlty', code_leb='l7', code_dub='d7'),
+       deposit_place = DepositPlace.objects.create(name='Dpp', code_leb='l8', code_dub='d8'),
+       quotation_place = QuotationPlace.objects.create(name='QP', code_leb='l9', code_dub='d9'),
+       asset_allocation = AssetAllocation.objects.create(name='AsA', code_leb='l10', code_dub='d10'),
+       group_for_ledgers = '4211',
+       general_ledger = '4649',
+       provider_code = 'pro',
+       provider_ratelist = RateListProvider.objects.create(name='rlp', code_leb=11, code_dub=11),
+       multiplier_for_online_prices = 1,
+       isin = 'isi',
+       fixing = True,
+       fix1 = '10:00', 
+       fix2 = '11:00',
+       mar_short_position = '11',
+       mar_lng_position = '12',
+       main_short_position ='13',
+       main_lng_position = '14',
+       moody = 'aaa',
+       fitch = 'bbb',
+       s_and_p = 'ccc'
+       )
+
+       with self.assertRaises(ValueError):
```

Comment:
are you sure this works? I don't see any `code_leb = None` or `code_dub = None` above

Reply:
sorry, while doing all my test, I forgot to re-put 'None'. I pushed the commit now

Reply:
Before your commit, `test_save_bond` was passing and `test_save_bond_2` was failing.
After your commit, `test_save_bond_2` is passing and `test_save_bond` is failing.